Author: Sean Finney <seanius@debian.org>
Description: Don't run git-clean via buildconf
 Calling buildconf indirectly invokes vcsclean, which calls the gitclean-work
 target of build/build.mk, which calls among other things git clean -X -f -d,
 which in turn nukes the quilt .pc directory making life quite difficult for
 us.
 .
 This patch doesn't need to go upstream, as they likely don't want to support
 having a patch system on top of their source.
--- php5.orig/build/build.mk
+++ php5/build/build.mk
@@ -75,6 +75,5 @@ gitclean-work:
 	@if (test ! -f '.git/info/exclude' || grep -s "git-ls-files" .git/info/exclude); then \
 		(echo "Rebuild .git/info/exclude" && echo '*.o' > .git/info/exclude && git svn propget svn:ignore | grep -v config.nice >> .git/info/exclude); \
 	fi; \
-	git clean -X -f -d;
 
 .PHONY: $(ALWAYS) snapshot
